src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -2388,31 +2534,29 @@ private void doNetworkReset(
 
             cleanupDecodeSlotIfNecessary();
 
-            doReset(network, routeId, initialId, traceId, authorization, EMPTY_OCTETS);
+            doReset(network, routeId, initialId, initialSeq, initialAck, bufferPool.slotCapacity(), traceId, authorization,
+                EMPTY_OCTETS);
         }
 
         private void doNetworkWindow(
             long traceId,
             long authorization,
-            int credit,
             int padding,
+            int maximum,
             long budgetId)
         {
-            assert credit > 0;
-
             state = AmqpState.openInitial(state);
 
-            initialBudget += credit;
-            doWindow(network, routeId, initialId, traceId, authorization, budgetId, credit, padding, 0);
+            doWindow(network, routeId, initialId,  initialSeq, initialAck, maximum, traceId, authorization, budgetId, padding, 0);
         }
 
         private void decodeNetworkIfNecessary(
             long traceId)
         {
             if (decodeSlot != NO_SLOT)
             {
-                final long authorization = 0L; // TODO
-                final long budgetId = 0L; // TODO
+                final long authorization = 0L;
+                final long budgetId = 0L;
```

Comment:
I think we should keep the `TODOs` here as the rationale for having them has not changed.